logging/bin/getlogs.py

```diff
@@ -135,10 +147,10 @@ def validate_input(dict):
             print("Given start date is after the end date.")
             exit()
         else:
-            dict['dateTimeStart'] = time.strftime("%Y-%m-%dT%H:%M:%S%z", time.gmtime(time.mktime(dict['dateTimeStart'])))
-            dict['dateTimeEnd'] = time.strftime("%Y-%m-%dT%H:%M:%S%z", time.gmtime(time.mktime(dict['dateTimeEnd'])))
+            dict['dateTimeStart'] = time.strftime("%Y-%m-%dT%H:%M:%S", time.gmtime(time.mktime(dict['dateTimeStart'])))
```

Comment:
small nit, but whats up with the formatting on this line? or does it just look like this in the PR?